backend/lib/plugins/error-handler.js

```diff
@@ -0,0 +1,43 @@
+const fp = require("fastify-plugin");
+const Sentry = require("@sentry/node");
+
+// Default Fastify error handler
+// https://github.com/fastify/fastify/blob/5bc406d4c6677131237f85b6ae69fe9e34d4b8dd/lib/context.js#L29
+function defaultErrorHandler(error, request, reply) {
+  if (reply.statusCode >= 500) {
+    reply.log.error(
+      { req: request, res: reply, err: error },
+      error && error.message,
+    );
+  } else if (reply.statusCode >= 400) {
+    reply.log.info({ res: reply, err: error }, error && error.message);
+  }
+  reply.send(error);
```

Comment:
this will put back the original default handler which was overridden by `fastify-sensible` and will not return all internal error server error info to the client which is something we probably want to avoid

Reply:
I've changed this to the fastify-sensible error handler.